providers/gov/usgs/harvester.py

```diff
@@ -1,5 +1,33 @@
+from furl import furl
+
 from share.harvest.harvester import Harvester
 
 
 class USGSHarvester(Harvester):
-    pass
+    url = 'https://pubs.er.usgs.gov/pubs-services/publication'
+
+    def do_harvest(self, start_date, end_date):
+        end_date = end_date.date()
+        start_date = start_date.date()
+
+        days_back = (end_date - start_date).days
```

Comment:
This a bit odd to do, if I run this harvester from 2014 - 2015 it would end up collecting data from 2016.
There should probably be a check that start date is today.